core/src/main/java/fr/sncf/osrd/stdcm/graph/STDCMPostProcessing.java

```diff
@@ -160,4 +163,41 @@ private static List<TrainStop> makeStops(List<Pathfinding.EdgeRange<STDCMEdge>>
         }
         return res;
     }
+
+    /** Builds the list of stops from OP */
+    private static List<TrainStop> makeOpStops(RawSignalingInfra infra, PathProperties trainPath) {
+        var operationalPoints = makeOperationalPoints(infra, trainPath);
+        var res = new ArrayList<TrainStop>();
+        for (var op: operationalPoints) {
+            res.add(new TrainStop(op.pathOffset, 0));
+        }
+        return res;
+    }
+
+    /** Sorts the stops on the path. When stops overlap, the user-defined one is kept. */
+    private static List<TrainStop> sortAndMergeStopsDuplicates(List<TrainStop> stops) {
+        stops.sort(Comparator.comparingDouble(st -> st.position));
+        var res = new ArrayList<TrainStop>();
+        TrainStop last = null;
+        for (var stop : stops) {
+            if (last != null && Math.abs(last.position - stop.position) < 0.001)
```

Comment:
When comparing positions, we wanna use `arePositionsEqual()` from now on. See `TrainPhysicsIntegrator.java`.